lib/types/column.ts

```diff
@@ -0,0 +1,18 @@
+export interface Column {
+  name: string;
+  type: string;
+  defaultValue: any | null;
```

Comment:
same as before, what's the advantage of explicit null over optional field?

Reply:
See before 🙂